jwt/api.py

```diff
@@ -25,6 +25,25 @@ def __init__(self, algorithms=None):
             if key not in self._valid_algs:
                 del self._algorithms[key]
 
+        if not options:
+            options = {}
+
+        self.default_options = {
+            'verify_signature': True,
+            'verify_exp': True,
+            'verify_nbf': True,
+            'verify_iat': True,
+            'verify_aud': True,
+        }
+
+        try:
+            merged_options = dict()
+            for k, v in self.default_options.items():
+                merged_options[k] = options.get(k, v)
```

Comment:
That doesn't bother me since those extra keys won't affect the behavior of the library. If anything, we may introduce future extensibility so custom options may make sense.

I think the code is a cleaner without the loop too.